classes/submission/Collector.php

```diff
@@ -434,6 +441,15 @@ public function getQueryBuilder(): Builder
                 ->whereIn('pc.category_id', $this->categoryIds);
         }
 
+        if ($this->isReviewedBy !== null) {
```

Comment:
I suppose the `TODO` might cover this, but this `if` is redundant. This is personal taste, but I find the `->when` formulations clearer to read when everything is written with them consistently, rather than mixing `if` with `->when`.

Reply:
Yes, this should be replaced in the whole query builder. Leaving for later.